src/status_im/navigation/events.cljs

```diff
@@ -119,7 +119,8 @@
   {:events [:show-bottom-sheet]}
   [{:keys [db]} content]
   (let [{:keys [sheets hide?]} (:bottom-sheet db)]
-    (rf/merge {:db (update-in db [:bottom-sheet :sheets] #(conj % content))}
+    (rf/merge {:db       (update-in db [:bottom-sheet :sheets] #(conj % content))
+               :dispatch [:dismiss-keyboard]}
```

Comment:
Will it always be the case that we'll want to hide the keyboard when showing the bottom sheet?

Reply:
I think the behavior there should be to focus on the input of the bottom sheet once it's open
We're dismissing the keyboard that was there _before_ rendering the bottom sheet.